ThingIFSDK/ThingIFSDK/OnboardWithVendorThingIDOptions.swift

```diff
@@ -41,3 +41,32 @@ public struct OnboardWithVendorThingIDOptions {
         self.layoutPosition = position
     }
 }
+
+extension OnboardWithVendorThingIDOptions: JsonObjectCompatible {
+
+    internal init(_ jsonObject: [String : Any]) throws {
+        // This method may not use so this method is not tested.
```

Comment:
@satoshikumano @yongpingchen 

I think we should divide the protocol because almost all case, a class requires only one method of the two methods.

Like KISS principle, implementing only required method makes code more clear and simpler.

Now I'm thinking about new name. Please wait:)